api-tests/specs/createTrialSite/createNewTrialSite.js

```diff
@@ -0,0 +1,104 @@
+const requests = require('../../data/createTrialSite/createNewTrialSite')
+const conf = require('../../config/conf')
+
```

Comment:
Declare the base uri as below and delete the 'before each' statement please. This will avoid conflict when running all the api tests at a go.

const endpointUri = ':83/sites';